LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
OK, was just trying to understand the reasoning. I guess I'm not feeling
one way or another, but I think AGPL was chosen in part on feedback from
our community (because can't you choose to close source a fork of Apache
licensed code, and continue forward development there?)

On Mon, Nov 23, 2015 at 3:04 PM, Joel Barciauskas <notifications@github.com>
wrote:

> In LICENSE
> <https://github.com/edx/xblock-lti-consumer/pull/1#discussion_r45653260>:
>
> > @@ -0,0 +1,202 @@
> > +
> > +                                 Apache License
>
> There are ways in which we think the AGPL was the wrong decision for the
> platform. I don't want to be the final word on this if this is a
> significant policy change, I thought we were moving towards Apache in
> general because the AGPL is overly limiting.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/xblock-lti-consumer/pull/1/files#r45653260>.
>